server/application/application.go

```diff
@@ -888,12 +888,12 @@ func (s *Server) Sync(ctx context.Context, syncReq *application.ApplicationSyncR
 		}
 		return a, err
 	}
-	active := proj.Spec.Maintenance.ActiveWindows()
-	match, _ := active.Match(a)
-	if match {
-		s.logEvent(a, ctx, argo.EventReasonOperationCompleted, fmt.Sprint("Cannot sync: Maintenance Windows are active"))
-		return a, err
+
+	if !proj.Spec.SyncWindows.Matches(a).CanSync(true) {
+		s.logEvent(a, ctx, argo.EventReasonOperationCompleted, fmt.Sprint("Cannot sync: Blocked by sync window"))
```

Comment:
Event `EventReasonOperationCompleted` should be created only sync operation completion. Please remove this line.

Reply:
Will do

Reply:
Done